src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+
+public class Report {
+    private Map<String, Integer> fruitQuantityMap;
+
+    public Report(Map<String, Integer> fruitQuantityMap) {
```

Comment:
Consider validating the 'fruitQuantityMap' parameter in the constructor to ensure it's not null, as passing a null value could lead to a NullPointerException later when the map is used.